src/common/UserAvatarWithPresence.js

```diff
@@ -1,13 +1,13 @@
 /* @flow strict-local */
 import React, { type ComponentType, PureComponent } from 'react';
```

Comment:
ESLint tells me we should remove `type ComponentType`.

Reply:
Ah thanks, indeed.